multitenancy-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/multitenancy/autoconfig/MultiTenantPersistentStreamMessageSource.java

```diff
@@ -0,0 +1,66 @@
+package org.axonframework.extensions.multitenancy.autoconfig;
+
+import io.axoniq.axonserver.connector.event.PersistentStreamProperties;
+import org.axonframework.axonserver.connector.AxonServerConfiguration;
+import org.axonframework.axonserver.connector.event.axon.PersistentStreamMessageSource;
+import org.axonframework.config.Configuration;
+import org.axonframework.extensions.multitenancy.components.MultiTenantAwareComponent;
+import org.axonframework.extensions.multitenancy.components.MultiTenantSubscribableMessageSource;
+import org.axonframework.extensions.multitenancy.components.TenantDescriptor;
+import org.axonframework.common.Registration;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ScheduledExecutorService;
+
+public class MultiTenantPersistentStreamMessageSource extends PersistentStreamMessageSource
```

Comment:
I don't think this is a Spring Boot auto-configuration specific class, right? 

I think it would be better placed in the `org.axonframework.extensions.multitenancy.components.eventstore` module, as the Event Store is a message source.